packages/telemetry/src/slog-to-otel.js

```diff
@@ -662,16 +871,17 @@ export const makeSlogToOtelKit = (tracer, overrideAttrs = {}) => {
         break;
       }
       case 'crank-start': {
-        const { message, ...crankAttrs } = slogAttrs;
-        const [name, messageAttrs, links] = extractMessageAttrs(message);
+        const { message, crankType, ...crankAttrs } = slogAttrs;
+        if (crankType === 'routing') {
+          break;
```

Comment:
Are we not reporting routing cranks at all then?

Reply:
Correct.  My recollection was that we decided that when we had our big discussion after I returned from vacation.  The rationale was that they have no duration and thus just add noise and complication to the analysis.